docs/vectorgrid-api-docs.html

```diff
@@ -399,7 +422,7 @@ <h3 id='symbolizer-method'>Methods</h3>
 <li>A set of <code>L.Path</code> options</li>
 <li>An array of sets of <code>L.Path</code> options</li>
 <li>A function that returns a set of <code>L.Path</code> options</li>
-<li>A function that returns an array of sets of <code>L.Path</code> options
+<li>A function that returns an array of sets of <code>L.Path</code> options for point, line, and polygon styles respectively
```

Comment:
Returning an array of style sets was listed as a valid approach before this proposed change. However, I don't know how those arrays were handled before. I therefore don't know whether I'm right to have amended this description of that option, or whether I should have created another &lt;li&gt; to describe the new functionality.

Reply:
The current behaviour is here: https://github.com/Leaflet/Leaflet.VectorGrid/blob/dfe447246682caeb3fb7de5d29e4edde1db92b9d/src/Leaflet.VectorGrid.js#L109

A symbolizer (based only on the feature's geometry and the tile's bounds) is instantiated. Then, the symbolizer "renders" each of the styles in the returned array into the tile's canvas or svg container.

Therefore, if the feature is a point, the styling function will receive `1` as the geometry type/dimension, and every style returned **will** be interpreted as styles for a `PointSymbolizer`. Idem for lines and polygons.